### PR TITLE
fix: remediate vulns

### PR DESCRIPTION
**Pull request by aarlaud**
- [x] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [ ] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
- [x] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)

### What this does

Fixes vulns

### CODE REVIEW

File: .dccache [hunk 1]
```diff
@@ -0,0 +1 @@
...onfig.js":[157,1629819233369.828,"44eb1cdcfbeaa045edb64c86e7a5c8116a4ff7755228aaeede3b1831d2b4233b"],"/Users/mathilderiou/Documents/code/snyk-prevent-gh-commit-status/src/lib/index.ts":[5023,1629822325286.1233,"9447a961e7b8006e3bd85fb2ee27d3232bbf1dee891820b1e6aaf7e129359dec"],"/Users/mathilderiou/Documents/code/snyk-prevent-gh-commit-status/src/lib/types.ts":[135,1629819233371.3647,"7dddefeff020b7b01ebe3d78e7f5e67fed1db1e439d0c46566c8913df64e6525"],"/Users/mathilderiou/Documents/code/snyk-prevent-gh-commit-status/test/lib/index.test.ts":[27638,1629822325286.6057,"1b8d211a31408d5e8850c8119675544357ab84f7efe9887dbaa088d9b86ed1cb"],"/Users/mathilderiou/Documents/code/snyk-prevent-gh-commit-status/src/lib/github/commitStatus.ts":[2386,1629822325285.1953,"9a2057f6c17c48ce96963a30dd22a3b4dd028b9163b760d3d1df4d34e171c034"],"/Users/mathilderiou/Documents/code/snyk-prevent-gh-commit-status/src/lib/github/prComments.ts":[5773,1629822325285.674,"e81cbf9a8b4767a767d2279233be2cca28a3b73e4f3964179e557cb60c415d8e"],"/Users/mathilderiou/Documents/code/snyk-prevent-gh-commit-status/src/lib/github/types.ts":[560,1629819233370.9846,"57e232ba46e5f48669d0159ed7c04f553eda5e1564c8c748f446f43946d072f7"]}
```
Comment (lili2311) [inline .dccache]: please delete the file & add to `.gitignore`

File: src/lib/index.ts [hunk 2]
```diff
@@ -44,7 +44,7 @@ const main = async () => {
 
     debug(`running snyk-prevent-gh-commit-status with org: ${ghOrg} repo: ${ghRepo} commit: ${ghSha} PRNumber: ${ghPRNumber} keepHistory: ${keepHistory} setPassIfNoBaselineFlag: ${setPassIfNoBaselineFlag} detailsLink: ${detailsLink}`)
 
-    const snykDeltaDebug = process.env.SNYK_DEBUG ? true : false; // process.argv.slice(2)[6] == 'debug' ? true : false;
+    const snykDeltaDebug = true; // process.argv.slice(2)[6] == 'debug' ? true : false;
```
Comment (lili2311) [inline src/lib/index.ts]: this is always in debug mode now?

File: .vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```
Comment (lili2311) [inline .vscode/launch.json]: are the comments needed?

File: src/lib/index.ts (hunk 2 above)
Comment (mathild3r) [reply src/lib/index.ts]: No, thought I had deleted that

File: .vscode/launch.json (hunk 3 above)
Comment (mathild3r) [reply .vscode/launch.json]: @aarlaud ? that was there when I started working on that branch

Comment (aarlaud) [reply .vscode/launch.json]: Drop it

File: test/fixtures/apitest-gomod-aggregated.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
+    "issues": [
```
Comment (aarlaud) [inline test/fixtures/apitest-gomod-aggregated.json:2]: That's expected to be empty?

**Review by mathild3r — APPROVED** [review]

**Pull request merged**